types/module-alias/module-alias-tests.ts

```diff
@@ -0,0 +1,3 @@
+import * as moduleAlias from 'module-alias';
```

Comment:
Can you switch this to an `import moduleAlias = require("module-alias")` and add a test using this as a function?

Reply:
commited requested changes in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/25911/commits/6891a455e9b34b7668ed1429604cad59fe485c0b